### ISSUE TITLE
Enable MBR in DIA-NN step

### ISSUE THREAD

**Issue by daichengxin** [bigbio/quantms#238]
### Description of feature

Now, due to make things as parallelisable as possible, Thus, step 2 consists of analysing each run separately with the in silico library generated in step 1.  so we should think about adding MBR in one big node . See details: https://github.com/bigbio/quantms/issues/164

**Comment by calliza** [comment]
Hi, this is marked as completed, but I don't see the --reanalyse parameter anywhere in the code. What was the strategy to implement MBR for parallelised DiaNN?

**Comment by ypriverol** [comment]
@calliza you are right. Let me re-open the issue.

**Comment by michaelsteidel86** [comment]
Is this still considered as an open issue? To my understanding the re-analysis step based on the empirical library actually is an MBR -analysis. Please correct me if I am wrong. So --re-analysis is only required if full analysis (1st and 2nd pass) are being in one step (non-parallelized)

**Comment by calizilla** [comment]
> Is this still considered as an open issue? To my understanding the re-analysis step based on the empirical library actually is an MBR -analysis. Please correct me if I am wrong. So --re-analysis is only required if full analysis (1st and 2nd pass) are being in one step (non-parallelized)

Yes I think you are correct, apologies for causing a re-open, I was new to DIA-NN at the time.

**Comment by jpfeuffer** [comment]
No worries! Good to clear that up for other users.
